### PR TITLE
Add years menu

### PR DESCRIPTION
**Pull request by jnth**
Add a new `Years` menu.

### CODE REVIEW

File: src/subsonic.ts [hunk 1]
```diff
@@ -1037,7 +1044,24 @@ export class Subsonic implements MusicService {
         .then(it => 
           it.find(station => station.id === id)!
         ),
-      
+      years: async () => {
+        const q: AlbumQuery = {
+          _index: 0,
+          _count: 100000,  // FIXME: better than this ?
```
Comment (jnth) [inline src/subsonic.ts:1050]: Is there a better way to do this ?

Comment (simojenki) [reply src/subsonic.ts:1050]: probably, however it's good enough for now.  There is an initiative to improve the subsonic API, part of that addresses this type of thing so I'll tidy this up when I get a chance to look into the new API some more.

**Comment by simojenki** [comment]
Other than the comment re: test for function this looks good. I've tested locally, works find for me against ND.

**Comment by jnth** [comment]
Is there any additional work needed in this pull request?

**Pull request merged**